django/db/models/base.py

```diff
@@ -1094,8 +1094,8 @@ def _prepare_related_fields_for_save(self, operation_name, fields=None):
                     if not field.remote_field.multiple:
                         field.remote_field.delete_cached_value(obj)
                     raise ValueError(
-                        "%s() prohibited to prevent data loss due to unsaved "
-                        "related object '%s'." % (operation_name, field.name)
+                        f"{operation_name}() prohibited to prevent data loss due to "
+                        f"unsaved related object '{field.name}'."
```

Comment:
Please revert this, it's not worth changing existing code.

Reply:
No worries, reverted